backend/src/lib/report-generator.js

```diff
@@ -346,6 +346,8 @@ async function prepAuditData(data) {
             username: collab.username || "undefined",
             firstname: collab.firstname || "undefined",
             lastname: collab.lastname || "undefined",
+            email: collab.email || "undefined",
+            email: collab.phone || "undefined",
```

Comment:
I feel like you probably meant phone here instead of double email

Reply:
Yes, thanks for the noting that out, I think I missed to save the changes in this file